sim/mcuboot-sys/build.rs

```diff
@@ -92,6 +91,7 @@ fn main() {
 
     walk_dir("../../boot").unwrap();
     walk_dir("../../ext/tinycrypt/lib/source").unwrap();
+    walk_dir("../../ext/mbedtls/src").unwrap();
```

Comment:
This shouldn't go into 'src' since it will miss header files. Just call
```
walk_dir("../../ext/mbedtls").unwrap()
```
The tinycrypt one is broken as well, but can be fixed in a different series.

Reply:
done